src/systems/menu_impl.cpp

```diff
@@ -46,41 +60,57 @@ static void pop_menu() {
     pad::reset_dir_repeat();
 }
 
+static bool show_hideable_widget(Widget* widget) {
+    HideableGroupWidget hideable = widget->hideable_group;
+    return hideable.show_if();
+}
+
```

Comment:
Can we remove this at this point?

Reply:
Yeah, I'll just replace all the references with the `show_if()` call.